broker/broker_utils/README.md

```diff
@@ -0,0 +1,9 @@
+# `pgb_broker_utils`
+
+- `pip install pgb_broker_utils`
+- `import broker_utils`
+
+Note that the schema maps are stored in a Cloud Storage bucket and are not packaged with this module.
```

Comment:
What implications does this have for future reproducibility?

Reply:
Good question that I hadn't thought specifically about yet... The schema map files are in the code repo so, for example, a tagged release will include the schema maps used in that version. Of course this begs the question of how we will facilitate reproducibility in general which I will continue to think about. I anticipate putting something in writing for the paper.